attributes/network/14_proof.adoc

```diff
@@ -0,0 +1,28 @@
+= Proof Attribute (#14)
+:cddl: ./cddl/
+
+This attribute allows server to deliver the proof of an object requested.
+The proof itself is a sequence of actions used to construct the state in which the object is embedded within the corresponding Merkle tree.
+
+== Attribute Argument
+
+This is a uint representing the hasher to be used in constructing proofs.
+
+* 0 - merk default hash (currently blake3)
+
+* 1 - TBD
+
+== Hash Scheme
+
+Each node contains a "kv hash", which is the hash of a prefix concatenanted with the nodes key and its value.
+The hash of the node is the hash of a distinct prefix, the kv hash, and the hash of each of the left and right child nodes.
+
+```
+kv_hash = H(0x00, key, value)
+node_hash = H(0x01, kv_hash, left_child_hash, right_child_hash)
+```
```

Comment:
We should move this piece to the other `adoc` document; it's not related to the attribute.

FYI, our `merk` fork doesn't contain the second preimage attack mitigation, i.e., the `0x00` and `0x01` prefixes are absent.